jwst/stpipe/step.py

```diff
@@ -602,7 +602,7 @@ def get_reference_file_model(self, input_file, reference_file_type):
 
         Returns
         -------
-        reference_file_model : jwst_lib.models.ModelBase instance
+        reference_file_model : jwst.datamodels.ModelBase instance
             A model to access the contents of the reference file.
         """
         from jwst import datamodels
```

Comment:
I'm pretty sure that import was already there and is not really part of this pull request.  I have no strong beliefs about which is best,  relative or absolute.  Would you like another pull request?